src/lib.rs

```diff
@@ -66,7 +68,7 @@ pub mod lib {
     pub use self::core::mem;
     pub use self::core::mem::ManuallyDrop;
     pub use self::core::ptr;
-
+    pub use core::any::Any;
     pub use self::core::f64;
```

Comment:
`self::core::any::Any` for consistency, and let's sort this block alphabetically while at it?